os/tools/configure.sh

```diff
@@ -153,7 +153,7 @@ fi
 if [ -r ${dest_config} ]; then
   if [ -r "${TOPDIR}/.version" ]; then
     echo "Already configured and compiled!"
-    echo "Do 'make distclean' and try again."
+    echo "Do './dbuild.sh distclean' and try again."
```

Comment:
what if one doesn't want to use docker build env

Reply:
Yeah, so we added "otherwise ...".